book/src/guides/hot_reload.md

```diff
@@ -1,18 +1,20 @@
 # Hot reload
 
-Freya supports Dioxus hot reload, this means you can update the `layout` and `styling` of your app on the fly, without having to compile any rust code.
+Freya supports Dioxus hot reload, which means that you can update the `layout` and `styling` of your app on the fly, without having to recompile your project.
 
 ## Setup
 
-Just before launching your app, you need to initialize the hot-reload context:
+Before launching your app, you need to initialize the hot-reload context:
 
 ```rust, no_run
-fn main() {
+use freya::prelude::*;
+use freya::hot_reload::FreyaCtx;
```

Comment:
I should definitely be exporting `FreyaCtx` in the prelude mmm, I'll change it, we can leave this like this for now